### ISSUE TITLE
Release a version for dashbaord

### ISSUE THREAD

**Issue by qqeasonchen** [apache/eventmesh-dashboard#37]
We need to release  a new version for dashboard when a milestone is ready.

**Comment by Pil0tXia** [comment]
Certainly. However, considering the limited number of active developers at present, there is a need to expedite the development progress further.
